nightingale_msgs/srv/RoomPoseLookup.srv

```diff
@@ -0,0 +1,8 @@
+string room
+string feature
+---
+int8 SUCCESS=0
```

Comment:
is this better than putting the return codes as a comment and having one int instead of 4?

Reply:
If you are asking if we can return a string with a return status message, that implementation also works but seems a bit excessive. If you are asking why there are 4 int8 fields for the return status message, these are just constants as shown here: https://wiki.ros.org/msg#Constants. The use case I am using is the autogenerated enum in Python (ie. RoomPoseLookup.ROOM_NOT_FOUND).

Reply:
i meant second and sure nw

Reply:
Just confirming for my understanding lines 4-6 are just defining constants, then line 7 is the actual field that data will get sent through?

Reply:
Correct